nupic/embodied/envs/real_robots/wrappers.py

```diff
@@ -89,45 +89,51 @@ class GoalWrapper(object):
     crop_obs: Specifies whether the retina image + desired & achieved
             goal should be cropped from 240x320x3 to 180x180x3 (mainly
             cuts off white space)
-    '''
+    """
+
     def __init__(self, env, crop_obs=False):
         self._env = env
         self.crop_obs = crop_obs
 
         if self.crop_obs:
-            obs_shape_1D = 180*180*3
+            obs_shape_1d = 180 * 180 * 3
         else:
-            obs_shape = env.observation_space['retina'].shape
-            obs_shape_1D = obs_shape[0]*obs_shape[1]*obs_shape[2]
+            obs_shape = env.observation_space["retina"].shape
+            obs_shape_1d = obs_shape[0] * obs_shape[1] * obs_shape[2]
 
-        self._env.action_space = self._env.action_space['joint_command']
+        self._env.action_space = self._env.action_space["joint_command"]
         # TODO: replace goal placeholders with actual goals
-        self._env.observation_space = spaces.Dict(dict(
-            desired_goal=spaces.Box(low=0, high=255.0, shape=[obs_shape_1D], dtype='float32'),
-            achieved_goal=spaces.Box(low=0, high=255.0, shape=[obs_shape_1D], dtype='float32'),
-            observation=spaces.Box(low=0, high=255.0,shape=[obs_shape_1D], dtype='float32')
-        ))
+        self._env.observation_space = spaces.Dict(
```

Comment:
Another piece of code reformatted with black, because the original one didn't fit the max line length of 88.